torch/cuda/nvtx.py

```diff
@@ -66,6 +66,20 @@ def range_end(range_id) -> None:
     _nvtx.rangeEnd(range_id)
 
 
+def device_range_start(msg) -> int:
+    """
+    TBD
```

Comment:
? add full docstring ?

Reply:
This depends on whether `RangeHandle` approach from above is the right way to go. I'll update the doc to cover the current implementation.

Reply:
I dont know a ton about nvtx.. not maybe @ezyang or @eqy might be able to weigh in

Reply:
Added some docs https://github.com/pytorch/pytorch/pull/130429/commits/6390c08720b9c2afe142e178c5861e7bcb644c92
Please, let me know if I can make it clearer 🙇